docs/REST API/rest-api.md

```diff
@@ -8881,7 +8881,7 @@ Get a list of all software.
       "software_package": {
         "name": "FirefoxInsall.pkg",
         "version": "125.6",
-        "self_service": true
+        "self_service": true,
```

Comment:
the final item in an object typically doesn't have the trailing comma. It was more correct before.